main.js

```diff
@@ -75,7 +75,7 @@ if (handleStartupEvent()) {
   console.log('OpenBazaar started on Windows...');
 }
 
-const serverPath = `${__dirname}${path.sep}..${path.sep}OpenBazaar-Server${path.sep}`;
+const serverPath = `${__dirname}${path.sep}..${path.sep}openbazaar-go${path.sep}`;
```

Comment:
@jjeffryes @hoffmabc 

Do you guys see any chance that someone using the stand-alone app (from source or otherwise) would have an openbazaar-go folder one directory up from where they are running the client?

I think it's unlikely, but if it did happen, then the client would think it's running the bundled app and some weird things could happen.

Would it be safer and reasonable do-able to have the bundled app install the server in a more uniquely named folder, like `OpenBazaar-Bundled-Server`?